### DISCUSSION TITLE
`uses: docker://namespace/name:version@sha256:2e…` reports "manifest unknown"

### DISCUSSION THREAD

**Discussion by l0b0** [community/community#74497] [Actions] [answered]
### Select Topic Area

Question

### Body

I'd like to run a Docker container "directly" with the `uses` directive, but it doesn't seem to be possible to pin it with a digest:

```
uses: docker://rhysd/actionlint:1.6.26@sha256:2eb91a78b5a19140be099c7b4262d298c2567f2a9f27e10ed2a4323c5bcface8
```

reports

> Error response from daemon: manifest for rhysd/actionlint:2eb91a78b5a19140be099c7b4262d298c2567f2a9f27e10ed2a4323c5bcface8 not found: manifest unknown: manifest unknown

I can `docker image pull rhysd/actionlint:1.6.26@sha256:2eb91a78b5a19140be099c7b4262d298c2567f2a9f27e10ed2a4323c5bcface8` locally. Is pinning with a digest just not supported?

**Comment by l0b0** [comment, marked as answer]
Actually, it's just not documented. It works fine. I was looking at the result of the wrong push.

**Reply by ettaboyle** [reply]
Thank you for responding with clarification & we're glad to see you found a resolution.
